index.html

```diff
@@ -22,18 +26,149 @@ <h3 class="poppins"><a href="#">My Logo</a></h3>
     <h1 class="poppins">I'm Victoria Glad to see you!</h1>
 
     <p class="poppins-regular">
-      I am a software developer. I can help you build a product, feature 
+      I'm a software developer! I can help you build a product, feature 
       or website. Look through some of my work and experience! If you like 
       what you see and have a project you need coded, do not hesitate to contact me.
     </p>
     <h3 class="poppins-regular">LET'S CONNECT</h3>
-      <ul>
-        <li><img class="a" src="./assets/twitter.svg"></li>
-        <li><img class="a" src="./assets/Linkedin icon.svg"></li>
-        <li><img class="a" src="./assets/M.svg"></li>
-        <li><img class="a" src="./assets/cat.svg"></li>
-        <li><img class="a" src="./assets/peace.svg"></li>
+      <ul class="social-media-icons">
+        <li class="a"><img  src="./assets/twitter.svg"></li>
+        <li class="a"><img  src="./assets/Linkedin icon.svg"></li>
+        <li class="a"><img  src="./assets/M.svg"></li>
+        <li class="a"><img  src="./assets/cat.svg"></li>
+        <li class="a"><img  src="./assets/peace.svg"></li>
       </ul>
   </section>
+  <br/>
+  <br/>
+
+  <section class="second-wrapper">
+    <div class="image-wrapper">
+      <img class="snapshoot" src="./assets/second-wrapper.png">
+    </div>
+   
+    <h2 class="poppins">Tonic</h2>
+    <ul class="poppins" id="about-ul">
+      <li class="about-li" id="canopy">CANOPY</li>
+      <li class="about-li">Back End Dev</li>
+      <li class="about-li">2015</li>
+    </ul>
+    <br/>
+    <p class="poppins-regular">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="poppins-regular" id="languages">
+      <li class="languages">html</li>
+      <li class="languages">css</li>
+      <li class="languages">javascript</li>
+    </ul>
+    <button class="poppins-regular">See Project</button>
+  </section>
+  <br/>
+  <br/>
```

Comment:
[OPTIONAL] : You can remove those `br` tag and give some `margin-bottom` to your `wrappers`.